flowauth/frontend/package.json

```diff
@@ -38,7 +38,7 @@
   ],
   "husky": {
     "hooks": {
-      "pre-commit": "pretty-quick --staged"
+      "pre-commit": "pretty-quick --staged --pattern \"**/*.*(js|jsx|json)\""
```

Comment:
Purely out of curiositoy, why the last `*` after the dot and before the `js/jsx/json` suffix?